src/platform/Tizen/WiFiManager.cpp

```diff
@@ -694,6 +917,26 @@ CHIP_ERROR WiFiManager::Disconnect(const char * ssid)
     return err;
 }
 
+CHIP_ERROR WiFiManager::StartWiFiScan(ByteSpan ssid, DeviceLayer::NetworkCommissioning::WiFiDriver::ScanCallback * callback)
```

Comment:
Where is the "ssid" argument used?

Reply:
The ssid will be used while doing specific scan. The code is updated now.